blocks/library/button/index.js

```diff
@@ -115,18 +115,18 @@ registerBlockType( 'core/button', {
 							checked={ !! clear }
 							onChange={ toggleClear }
 						/>
-						<PanelBody title={ __( 'Button Background Color' ) }>
+						<PanelColor title={ __( 'Button Background' ) } colorValue={ color } key="panel-color-button-1">
 							<ColorPalette
 								value={ color }
 								onChange={ ( colorValue ) => setAttributes( { color: colorValue } ) }
 							/>
-						</PanelBody>
-						<PanelBody title={ __( 'Button Text Color' ) }>
+						</PanelColor>
+						<PanelColor title={ __( 'Button Text' ) } colorValue={ textColor } key="panel-color-button-2">
```

Comment:
Removing the word "Color" doesn't help users understand what this control does. "Button Text" could be the text itself, or the font, or the text size? I'd just use `Background Color` and `Text Color` for both the paragraph and the button block.